### PR TITLE
Forward port system requirements differences in 10.3

### PR DESCRIPTION
**Pull request by settermjd**

### CODE REVIEW

**Comment by HanaGemela** [comment]
@mmattel @settermjd 10.3 shows the wrong heading for Desktop (and seems we lost Mobile completely).  This PR and https://github.com/owncloud/docs/pull/2333 are back porting this bug into master and 10.4. 
Shouldn't it be the other way round? Backport the correct information to 10.3?

File: modules/admin_manual/pages/installation/system_requirements.adoc [hunk 1]
```diff
@@ -97,21 +79,13 @@ with a MySQL or MariaDB database:
...
 * `READ COMMITED` transaction isolation level (See: 
 xref:configuration/database/linux_database_configuration.adoc#mysql-mariadb-read-commited-transaction-isolation-level[MySQL / MariaDB `READ COMMITED` transaction isolation level])
-====
-
 
-=== Mobile
+== Memory Requirements
 
-* iOS 9.0+
-* Android 4.4+
```
Comment (phil-davis) [inline modules/admin_manual/pages/installation/system_requirements.adoc]: These seem to have disappeared. Is that correct? Are they somewhere else, in some other document?

File: modules/admin_manual/pages/installation/system_requirements.adoc [hunk 2]
```diff
@@ -120,21 +94,34 @@ xref:configuration/database/linux_database_configuration.adoc#mysql-mariadb-read
 * Xen
 * KVM
 
-=== Desktop
+=== Mobile OS
```
Comment (phil-davis) [inline modules/admin_manual/pages/installation/system_requirements.adoc]: Why did this heading change to "Mobile OS"?
The items under here look like Desktop OS.

**Comment by settermjd** [comment]
> @mmattel @settermjd 10.3 shows the wrong heading for Desktop (and seems we lost Mobile completely). This PR and #2333 are back porting this bug into master and 10.4.
> Shouldn't it be the other way round? Backport the correct information to 10.3?

Thanks for pointing out the mistakes in this PR. I'll get them corrected.

File: modules/admin_manual/pages/installation/system_requirements.adoc (hunk 2 above)
Comment (settermjd) [reply modules/admin_manual/pages/installation/system_requirements.adoc]: Not sure how that slipped through. Will correct.

**Comment by settermjd** [comment]
> We need to somehow do a PR that "resets" this in `master` to whatever it is that everybody wants.
> It got out-of-sync because of people doing edits in other branches, and then those edits not being promptly forward-ported to master. I wonder how we can make the process better in future? It is much to easy to get "random" edits merged in branches and then the forward-backward-porting gets forgotten.

What has worked very well is following the basic, if not all that fascinating, branch and backport workflow, then content changes have worked very well, almost without issue; that being branch off of master, and backport to the two most recent branches when the branch is merged. It seems that changes through the "_Edit this page_" link are the challenge. Yet, even so, it should still work. 

As I write this, I'm wondering if what the problem is not where the branches are created ("Edit this page", or by branching off of master) but rather the review process. I know that before it became mandatory to have at least one reviewer for each feature branch and for the build to pass there were a number of branches that got through with errors in them.

To the best of my knowledge, since those rules have been applied, the quality of the content and the backporting process has improved immeasurably. So, it would be helpful to know when the changes were made, to see if they came before or after the branch rules change.

**Comment by phil-davis** [comment]
I think one of the challenges is when a PR happens to a branch (e.g. 10.3). "Somebody" has to recognise/realise that it went to 10.3 branch and then initiate forward-porting to master, 10.4 (10.5... whatever branches exist)

It seems to work for PRs to master - if the originator does not do it then @settermjd or @mmattel seem to notice and make the backports to 10.4 and 10.3.

But with the "edit this branch" PRs to 10.3... then the forward-porting gets missed/forgotten.

**Comment by settermjd** [comment]
> I think one of the challenges is when a PR happens to a branch (e.g. 10.3). "Somebody" has to recognise/realise that it went to 10.3 branch and then initiate forward-porting to master, 10.4 (10.5... whatever branches exist)
> 
> It seems to work for PRs to master - if the originator does not do it then @settermjd or @mmattel seem to notice and make the backports to 10.4 and 10.3.
> 
> But with the "edit this branch" PRs to 10.3... then the forward-porting gets missed/forgotten.

Sounds fair. Do you have an example of a recent one that I can investigate further?

**Comment by settermjd** [comment]
@phil-davis, I reworked the page in 8cdab0d. Please let me know if I've done this in error, or whether it's correct decision to make.

File: modules/admin_manual/pages/installation/system_requirements.adoc [hunk 3]
```diff
@@ -7,134 +7,115 @@
...
 
-| Database
+|Database
 |
-* MySQL 8+ or MariaDB 10+ (RECOMMENDED)
-* Oracle 12g
-* PostgreSQL 9 (versions 10 and above are not _yet_ supported)
-* SQLite (NOT FOR PRODUCTION)
+* MySQL 8+ or MariaDB 10+ (*Recommended*)
+* Oracle 11 and 12
```
Comment (phil-davis) [inline modules/admin_manual/pages/installation/system_requirements.adoc]: This seems to be a real change. It already says Oracle 11 and 12 in 10.4 and 10.3 branches, so this is just making it consistent.

**Pull request merged**